scheduler/scheduler.go

```diff
@@ -156,7 +156,16 @@ func (s *Scheduler) Schedule(ctx context.Context) error {
 					"expected_duration_span": res.schedulerEvent.ExpectedDuration.String(),
 					"span":     time.Since(distroStartTime).String(),
```

Comment:
I believe one of our distros has an old go that doesn't support time.Since().

Also, can call time.Since() once, store the result in a variable, and use the variable here and on the next line, so that two fields represent the same duration?